examples/platforms/nrf52840/README.md

```diff
@@ -24,53 +24,50 @@ $ ./script/bootstrap
 
 ## Building the examples
 
+ To build the examples, run the following command in Bash:
+
 ```bash
 $ cd <path-to-openthread>
 $ ./bootstrap
 $ make -f examples/Makefile-nrf52840
 ```
 
 After a successful build, the `elf` files can be found in
-`<path-to-openthread>/output/nrf52840/bin`.  
-You can convert them to hex using arm-none-eabi-objcopy`:
+`<path-to-openthread>/output/nrf52840/bin`.  You can convert them to `hex`
+files using `arm-none-eabi-objcopy`:
+
 ```bash
 $ arm-none-eabi-objcopy -O ihex ot-cli-ftd ot-cli-ftd.hex
 ```
 
-## Native USB support
-
-You can build the libraries with support for native USB CDC ACM as a serial transport.
-To do so, build the libraries with the following parameter:
+You can build the libraries with support for the native USB CDC ACM as a serial transport.
+To do so, build the firmware with the following parameter:
 ```
 $ make -f examples/Makefile-nrf52840 USB=1
 ```
 
 Note, that if Windows 7 or earlier is used, an additional USB CDC driver has to be loaded.
```

Comment:
This was not fixed. Edit:

If you are using Windows 7 or earlier, you must load an additional USB CDC driver.
The driver can be found in third_party/NordicSemiconductor/libraries/usb/nordic_cdc_acm_example.inf

Reply:
fixed